### ISSUE TITLE
Change cs-desktop color.

### ISSUE THREAD

**Issue by Initu-Castilhos** [linuxmint/mint-y-icons#324]
As the icon was recently changed, it now resembles the cs-panel, to make it more different it is better to change the color of the cs-desktop icon like the example below:
![cs-desktop](https://user-images.githubusercontent.com/59381215/196003701-d9d9c2bc-3183-48b2-ac1b-e60a5ad88bd4.svg)

My argument for using Mint's default highlight color is that Mint-X's legacy cs-desktop system default icon is green and Mint-Y's system settings icon is green too and is used in all variants of Colors.
![user-desktop](https://user-images.githubusercontent.com/59381215/196003807-932ddbaf-79d6-43c0-8e04-c98874a8d840.svg)
![cs-general](https://user-images.githubusercontent.com/59381215/196003817-f308e929-138a-4873-a607-fd508094eecf.png)

**Comment by PHPeon** [comment]
I would unironically love this but it's ultimately Sir Lefèbvre's decision, i myself had several suggestions turned down

**Comment by clefebvre** [comment]
Yes, it's now identical to cs-panel, and that's an issue. It needs to be different than user-desktop also. If you search for Desktop in the menu, you see both the place and the settings.

What if we simply removed the black panel bar in cs-desktop? After all, the icon for Backgrounds doesn't have one, and it too represents the desktop somehow.
